src/switchroot/ostree-mount-util.h

```diff
@@ -106,6 +120,69 @@ read_proc_cmdline_ostree (void)
   return ret;
 }
 
+static inline char*
+gen_ostree_target (const char* prepend)
```

Comment:
This function really really wants a documentation comment describing what it does.

Reply:
Will add to give context